Scripts/script-findIncidentsWithIndicator.yml

```diff
@@ -0,0 +1,40 @@
+commonfields:
+  id: findIncidentsWithIndicator
+  version: 37
+name: findIncidentsWithIndicator
+script: |-
+  var res = executeCommand("getIncidents", {"query": "indicator.value:"+args.indicator});
```

Comment:
You need to surround the value with quotes for cases when you will have a space in it
maybe it should be with brackets as well, please check the syntax